apps/dashboard/src/components/primitives/control-input/variable-plugin/variable-pill-widget.ts

```diff
@@ -25,11 +25,75 @@ export class VariablePillWidget extends WidgetType {
     };
   }
 
+  createBeforeStyles(): CSSProperties {
```

Comment:
This `VariablePillWidget` is the component that represents the pill in the CodeMirror as the extension.
My first idea was to reuse the same component in the Email editor (Maily), and embed it in the react component, but later, I decided not too, because it's easier to control the react pill component `VariablePill`, than playing with DOM elements.
So I left the refactored styles and code, and IMO it's better to keep the styles close.